owncloudApp/src/main/java/com/owncloud/android/ui/adapter/FileListListAdapter.java

```diff
@@ -69,17 +71,19 @@
  */
 public class FileListListAdapter extends BaseAdapter implements ListAdapter {
 
-    private Context mContext;
+    private final Context mContext;
     private Vector<OCFile> mImmutableFilesList = null; // List containing the database files, doesn't change with search
     private Vector<OCFile> mFiles = null; // List that can be changed when using search
-    private boolean mJustFolders;
-    private boolean mOnlyAvailableOffline;
-    private boolean mSharedByLinkFiles;
-    private boolean mFolderPicker;
+    private final boolean mJustFolders;
+    private final boolean mOnlyAvailableOffline;
+    private final boolean mSharedByLinkFiles;
+    private final boolean mFolderPicker;
 
     private FileDataStorageManager mStorageManager;
     private Account mAccount;
-    private ComponentsGetter mTransferServiceGetter;
+    private final ComponentsGetter mTransferServiceGetter;
+
+    private static final String SHOW_HIDDEN_FILES = "show_hidden_files";
```

Comment:
I think we already created a constant: PREF_SHOW_HIDDEN_FILES, right? We don't need to create another one, don't you think?

Reply:
done